doc/index.rst

```diff
@@ -111,16 +111,24 @@ Documentation
 * :doc:`how_to_docs`
 * :doc:`introduction`
 * :doc:`installation`
+* :doc:`how_to_start`
 * :doc:`tutorial`
+* :doc:`data_workflow`
+* :doc:`how_to_customize`
+* :doc:`how_to_validate`
 
 .. toctree::
    :hidden:
    :maxdepth: 2
    :caption: Getting Started
 
    introduction
+   data_workflow
    installation
```

Comment:
May it be better to locate the data_workflow after the installation?

Reply:
Done :)